package/MDAnalysis/coordinates/base.py

```diff
@@ -2006,3 +2074,26 @@ def close(self):
         # self.filename. Explicitly setting it to the null action in case
         # the IOBase.close method is ever changed from that.
         pass
+    
+    def add_transformations(self, *transformations):
+        """ Adds all the transformations to be applied to the trajectory.
+        Overrides :meth:`~MDAnalysis.coordinates.base.ProtoReader.add_transformations`
```

Comment:
this isn't user documentation, so should be a comment.  Ie this isn't helpful when I'm in a ipython repr trying to figure out how to add transformations :)